test/spec_utils.rb

```diff
@@ -133,6 +133,12 @@ def assert_nested_query(exp, act)
     Rack::Utils.parse_nested_query("foo#{"[a]" * (len - 1)}=bar")
   end
 
+  # ParamsTooDeepError was introduced in the middle of 2.2 releases
+  # and this test is here to ensure backwards compatibility
+  it "ParamsTooDeepError is inherited from originally used RangeError" do
+    Rack::QueryParser::ParamsTooDeepError.ancestors.must_include(RangeError)
```

Comment:
```suggestion
    assert Rack::QueryParser::ParamsTooDeepError < RangeError
```

https://ruby-doc.org/core-3.1.1/Module.html#method-i-3C

?

Reply:
I'm trying to follow the rest of the file where "spec" assertions are used.

Reply:
I don't mind too much about this, just wondered if it was an easier way to express the relationship.

Reply:
What about `(Rack::QueryParser::ParamsTooDeepError < RangeError).must_equal(true)`?

Reply:
I'm fine with that if that's the best, totally up to you.

Reply:
updated